src/SWP/Bundle/ContentBundle/Tests/Controller/RoutesControllerTest.php

```diff
@@ -63,6 +63,26 @@ public function testCreateContentRoutesApi()
         $this->assertEquals('{"id":"\/swp\/default\/routes\/simple-test-route","content":null,"static_prefix":null,"variable_pattern":null,"name":"simple-test-route","children":[],"id_prefix":"\/swp\/default\/routes","_links":{"self":{"href":"\/api\/v1\/content\/routes\/\/simple-test-route"}}}', $client->getResponse()->getContent());
     }
 
+    public function testCreateContentRoutesApi()
```

Comment:
shouldn't the functionname refer to static pages somehow? it will be more obvious imo